databricks_cli/unity_catalog/delta_sharing_cli.py

```diff
@@ -0,0 +1,484 @@
+from databricks_cli.unity_catalog.api import UnityCatalogApi
+from databricks_cli.unity_catalog.utils import mc_pretty_format
+from databricks_cli.utils import eat_exceptions, CONTEXT_SETTINGS, json_cli_base
+
+
+##############  Share Commands  ##############
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Create a new share.')
+@click.option('--name', required=True, help='Name of new share.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def create_share_cli(api_client, name):
+    """
+    Create a new share.
+
+    Calls the 'createShare' RPC endpoint of the Unity Catalog service.
+    Returns the ShareInfo for the newly-created share.
+
+    """
+    share_json = UnityCatalogApi(api_client).create_share(name)
+    click.echo(mc_pretty_format(share_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List shares.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_shares_cli(api_client):
+    """
+    List shares.
+
+    Calls the 'listShares' RPC endpoint of the Unity Catalog service.
+    Returns array of ShareInfos.
+
+    """
+    shares_json = UnityCatalogApi(api_client).list_shares()
+    click.echo(mc_pretty_format(shares_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Get a share.')
+@click.option('--name', required=True,
+              help='Name of the share to get.')
+@click.option('--include-shared-data', default=True,
+              help='Whether to include shared data in the response.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def get_share_cli(api_client, name, include_shared_data):
+    """
+    Get a share.
+
+    Calls the 'getShare' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    share_json = UnityCatalogApi(api_client).get_share(name, include_shared_data)
+    click.echo(mc_pretty_format(share_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='List permissions on a share.')
+@click.option('--name', required=True,
+              help='Name of the share to list permissions on.')
+@debug_option
+@profile_option
+@eat_exceptions
+@provide_api_client
+def list_share_permissions_cli(api_client, name):
+    """
+    List permissions on a share.
+
+    Calls the 'getSharePermissions' RPC endpoint of the Unity Catalog service.
+    Returns nothing.
+
+    """
+    perms_json = UnityCatalogApi(api_client).list_share_permissions(name)
+    click.echo(mc_pretty_format(perms_json))
+
+
+@click.command(context_settings=CONTEXT_SETTINGS,
+               short_help='Update permissions on a share.')
+@click.option('--name', required=True,
+              help='Name of the share whose permissions are updated.')
+@click.option('--json-file', default=None, type=click.Path(),
+              help='File containing JSON request to POST.')
```

Comment:
Following other cli implementation, include the URL.

```suggestion
              help='File containing JSON request to POST to [replace with the post URL].')
```

Reply:
Addressed in #471.